app/src/main/java/org/onionshare/android/tor/TorManager.kt

```diff
@@ -87,12 +102,23 @@ class TorManager @Inject constructor(
         }
     }
 
+    private val obfs4Bridges: List<String> = getResourceLines(R.raw.obfs4)
+    private val meekBridges: List<String> = getResourceLines(R.raw.meek)
+    private val snowflakeBridges: List<String>
+        get() = getResourceLines(R.raw.snowflake).map { "$it $snowflakeParams" }
+    private val snowflakeParams: String by lazy {
```

Comment:
Kotlin noob question: why use `by lazy` for this val and `get()` for the other vals? Both would be evaluated when the field's accessed, wouldn't they?

Reply:
Right. AFAIK the difference is that lazy keeps the value, so the "getter" is only called once which `get()` executes the getter code every time it is called.